docs/monitoring.md

```diff
@@ -40,6 +40,100 @@ To view the web UI after the fact, set `spark.eventLog.enabled` to true before s
 application. This configures Spark to log Spark events that encode the information displayed
 in the UI to persisted storage.
 
+## Web UI Tabs
+The web UI provides an overview of the Spark cluster and is composed of following tabs:
+
+### Jobs Tab
+The Jobs tab displays a summary page of all jobs in the Spark application and a detailed page
```

Comment:
Instead doing everything in a huge "Monitoring and Instrumentation", could you create a separate page for each one and add the links to these contents? In the future, we can adding more descriptions in these pages.